src/Analyzer.PowerShellRuleEngine/PSRuleHostContext.cs

```diff
@@ -41,6 +41,12 @@ public PSRuleHostContext(TemplateContext templateContext, ILogger logger = null)
                 : new JsonSourceLocationResolver(templateContext);
         }
 
+        /// <inheritdoc/>
+        public override ActionPreference GetPreferenceVariable(string variableName)
+        {
+            return variableName.Equals("VerbosePreference", System.StringComparison.OrdinalIgnoreCase) ? ActionPreference.Continue : base.GetPreferenceVariable(variableName);
```

Comment:
Not sure how this method is used, but consider adding null-safety here.

Reply:
static method works for this, has null checks

Reply:
Also, these methods enable PSRule logging with verbose option. Helped me to debug issue with PSRule with Bernie.